src/input.js

```diff
@@ -131,6 +139,20 @@ const toInquirerQuestion = curry((sources, [name, config]) => {
         | merge(__, { name }), component(config.type));
 });
 
+const readFile = promisify(fs.readFile);
+
+/**
+ * Opens the provided files and returns them as node buffers
+ *
+ * @param {String[]} fields
+ * @param {Object} answers
+ * @return {Promise}
+ */
+const openFiles = fields =>
+    mapObjIndexed((value, key) => value
```

Comment:
What `key` and `value` means here?

Reply:
Nice one! I will rename to twhat they really mean!

Reply:
`value` is the value of the `param`, and `key` will now become `param`